src/AgedBrieUpdateor.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace GildedRose;
+
+class AgedBrieUpdateor implements ItemConnector
+{
+    private int $quality, $sellIn;
+    private string $name;
+
+    public function __construct(string $name, int $quality, int $sellIn)
```

Comment:
niepotrzebne kopiuj-wklej w każdej klasie to samo

Reply:
Fixed